netlify.toml

```diff
@@ -0,0 +1,31 @@
+# Settings in the [build] context are global and are applied to all contexts
+# unless otherwise overridden by more specific contexts.
+# More info at https://docs.netlify.com/configure-builds/file-based-configuration/
+[build]
+  # Directory to change to before starting a build relative to this file.
+  # This is where we will look for package.json/.nvmrc/etc.
+  # If not set, defaults to the root directory.
+  base = "/webui/react/"
+
+  # Directory that contains the deploy-ready HTML files and assets generated by
+  # the build. This is relative to the base directory if one has been set, or the
+  # root directory if a base has not been set. This sample publishes the
+  # directory located at the absolute path "root/project/build-output"
+  publish = "build/"
+
+  # Default build command.
+  command = "make get-deps build"
+
+  environment = { PUBLIC_URL = "", SERVER_ADDRESS = "https://detai.ml/fixed" }
```

Comment:
question: I haven't seen this domain before -- is this one of Determined's?

Reply:
Yes. it's a proxy I'll chat with @sean-adler tomorrow to figure out the best way to serve a cluster to back the previews

Reply:
I'll update it to the new address once @mackrorysd has the new cluster up and running.